src/lib/modules/console_listener/index.js

```diff
@@ -80,6 +80,21 @@ class ConsoleListener {
 
   _listenForLogRequests() {
     if (this.ipc.ipcRole !== 'server') return;
+    this.events.on('deploy:contract:receipt', receipt => {
+      if(receipt === undefined) return;
```

Comment:
```suggestion
      if(!receipt) return;
```
It's prettier :)

Reply:
Also, another way to handle this would be to only emit when `receipt` is truthy:

```
if (receipt) {
   // this emit here
}
```

While we're at it, is there a scenario where `receipt` is not defined? Because I think `deploy:contract:receipt` will only emit with a receipt.. So we might as well drop this check all together.